Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  OdometerView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Mikaela Caron on 10/11/23.
+//
+
+import SwiftUI
+
+struct OdometerView: View {
+    @Environment(ActionService.self) var actionService
+    @Environment(\.scenePhase) var scenePhase
+    @Bindable private var viewModel: OdometerViewModel
+    @State private var isShowingEditView = false
+    @State private var isShowingAddView = false
+    @State private var selectedOdometerReading: OdometerReading?
```

Comment:
These could all be moved into the view model

Reply:
fixed in https://github.com/mikaelacaron/Basic-Car-Maintenance/pull/175/commits/ee13fdd7a3edd65597172bd2be3c59c301780683